agent/agent.go

```diff
@@ -157,10 +157,10 @@ func (r *Agent) Run(recipe recipe.Recipe) (run Run) {
 
 	success := run.Error == nil
 	durationInMs := getDuration()
-	r.monitor.RecordRun(recipe, durationInMs, success)
-
+	dataCount := 0
+	r.monitor.RecordRun(recipe, durationInMs, success, dataCount)
 	if success {
-		r.logger.Info("done running recipe", "recipe", recipe.Name, "duration_ms", durationInMs)
+		r.logger.Info("done running recipe", "recipe", recipe.Name, "duration_ms", durationInMs, "Data Count", dataCount)
```

Comment:
let's try to have the same format, instead of `"Data Count"`, we can use `"data_count"`.